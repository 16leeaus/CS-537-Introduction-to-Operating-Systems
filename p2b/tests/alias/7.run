./mysh tests/7.in | sort --ignore-case
