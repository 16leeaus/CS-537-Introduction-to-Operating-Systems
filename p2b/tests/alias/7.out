alias
alias
alias
alias la /bin/ls -l -a
alias ll /bin/ls -l
alias ls /bin/ls
alias psh /usr/bin/ps --help
alias whoami /usr/bin/whoami
exit
la /bin/ls -l -a
la /bin/ls -l -a
ll /bin/ls -l
ls /bin/ls
psh /usr/bin/ps --help
psh /usr/bin/ps --help
unalias la
unalias ll
unalias ls
unalias psh
unalias whoami
whoami /usr/bin/whoami
